api/src/main/java/org/openmrs/DrugOrder.java

```diff
@@ -14,58 +14,60 @@
 
 /**
  * DrugOrder
- * 
+ *
```

Comment:
My assumption is that if you build the project before issuing the committing then these format changes should get weeded out